internal/cmd/auth_createapitokens.go

```
@@ -27,7 +29,14 @@ var createApiTokensCmd = &cobra.Command{
 			return err
 		}
 
-		tokenName := args[0]
+		tokenName := strings.TrimSpace(args[0])
+
+		validNameRegex := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
+
+		if !validNameRegex.MatchString(tokenName) {
+			fmt.Println("Invalid token name, creation aborted.")
```

Please return an error to interrupt the program execution here.